decidim-admin/app/commands/decidim/admin/update_help_sections.rb

```diff
@@ -5,22 +5,36 @@ module Admin
     # This command gets called when a content block is updated from the admin
     # panel.
     class UpdateHelpSections < Decidim::Command
-      def initialize(form, organization)
+      def initialize(form, organization, user)
         @form = form
         @organization = organization
+        @user = user
       end
 
       def call
         return broadcast(:invalid) unless @form.valid?
 
         ActiveRecord::Base.transaction do
           @form.sections.each do |section|
-            ContextualHelpSection.set_content(@organization, section.id, section.content)
+            next unless content_has_changed(section)
+
+            Decidim.traceability.perform_action!("update", ContextualHelpSection, @user, { "resource" => { "title" => section.id.humanize } }) do
+              ContextualHelpSection.set_content(@organization, section.id, section.content)
+              ContextualHelpSection.find_by(organization: @organization, section_id: section.id)
+            end
           end
         end
 
         broadcast(:ok)
       end
+
+      private
+
+      def content_has_changed(section)
```

Comment:
```suggestion
      def content_has_changed?(section)
```
As it returns a boolean, I think it'd be a bit more idiomatic to add the question mark sign at the end of the method name.